config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+
+  root to: 'works#top_works'
+
+  resources :votes, only: [:index, :new, :create]
```

Comment:
I suggest nesting this under works, and you only need to create votes, not list them or have a new form.

```ruby
resources :works do
  resources :votes, only: [:create]
end
```